README.md

```diff
@@ -23,86 +23,116 @@
 
 [![Short demo](./logo/demo.gif)](https://youtu.be/_c4U6ffumw8)
 
+## Welcome to the Wing Language! :wave:
 
-# Welcome! :wave:
+Introducing **Wing**, an open-source, statically-typed, cloud-oriented programming language that is ready to give your cloud development a lift! 💨
+We're here to make your journey through the skies of AWS and other cloud providers more enjoyable and productive. 😎
 
-**Wing** is a [cloud-oriented programming language]. It is a modern,
-object-oriented, and strongly-typed language. Most programming languages think
-about computers as machines. In Wing, **_the cloud is the computer_**.
+Created by [Elad Ben-Israel](https://github.com/eladb), the mastermind behind the [AWS CDK](https://github.com/aws/aws-cdk), Wing is designed to make your cloud development soar! 🦅
 
-Wing applications compile to Terraform and JavaScript that are ready to deploy
-to your favorite cloud provider, and can also be tested in your local
-environment using the [Wing Console](https://docs.winglang.io/getting-started/console).
+## What's so special about Wing? 🤔
 
-[cloud-oriented programming language]: https://docs.winglang.io/#what-is-a-cloud-oriented-language
+Wing takes a unique approach to cloud development - instead of thinking about computers as individuals machines 💻, it treats the ***entire cloud as the computer***. ☁️
+By abstracting the cloud, Wing allows anyone building cloud applications to focus on their business logic and choose the target cloud at compile time.
+
+The result? While your main code is written in Wing, the compilation artifacts are JavaScript and Terraform (with more provisioning engines on the way), meaning Wing can fit seamlessly into your existing stack! 🎯
+
+In addition, Wing provides a built-in local simulator, and an observability & debugging [console](https://docs.winglang.io/getting-started/console), making it easier for you to reduce cognitive load and context switching, enabling you to stay in your creative flow. 🎨
```

Comment:
> 🎨

Maybe I'm old, but the "emoji related to a word used in the previous sentence" thing should probably be kept in places where having an additional eye-catcher doesn't hurt (like headers)

Reply:
I'm down to try without them - updated